model/tensorflow/dffml_model_tensorflow/dnnc.py

```diff
@@ -23,6 +24,24 @@
 from dffml.model.model import ModelContext, Model, ModelNotTrained
 
 
+@dataclass
+class BaseConfig:
```

Comment:
`BaseConfig` is used other places, we should probably name it something more specific

```suggestion
class TensorflowBaseConfig:
```